ratatui-core/src/buffer/cell.rs

```diff
@@ -61,6 +62,13 @@ impl Cell {
         self.symbol.as_str()
     }
 
+    /// Merge the symbol of the cell with the one already on the cell.
+    pub fn merge_symbol(&mut self, symbol: &str, strategy: MergeStrategy) -> &mut Self {
+        let next = strategy.merge(self.symbol(), symbol);
+        self.symbol = CompactString::new(next);
```

Comment:
nit: `merged` sounds more correct than `next` here, WDYT?

```suggestion
        let merged = strategy.merge(self.symbol(), symbol);
        self.symbol = CompactString::new(merged);
```